detekt/wrapper/src/main/kotlin/io/buildfoundation/bazel/rulesdetekt/wrapper/WorkerExecutable.kt

```diff
@@ -0,0 +1,27 @@
+package io.buildfoundation.bazel.rulesdetekt.wrapper
+
+import bazel.worker.WorkerProtocol.WorkRequest
+import bazel.worker.WorkerProtocol.WorkResponse
+
+internal interface WorkerExecutable {
+
+    fun execute(request: WorkRequest): WorkResponse
+
+    class Impl(private val executable: Executable) : WorkerExecutable {
+
+        override fun execute(request: WorkRequest): WorkResponse {
```

Comment:
If you're making it reactive, maybe return type here should be `Single<WorkResponse>`?